fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/export/data/CheckPointUserData.java

```diff
@@ -1,18 +1,17 @@
 /*
- * (C) Copyright IBM Corp. 2019, 2020
+ * (C) Copyright IBM Corp. 2019, 2021
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package com.ibm.fhir.jbatch.bulkdata.export.common;
+package com.ibm.fhir.bulkdata.jbatch.export.data;
 
 import java.util.List;
 
 import com.ibm.cloud.objectstorage.services.s3.model.PartETag;
 
 /**
- * Bulk export Chunk implementation - job check point data.
- *
+ * Bulk Export Job Checkpoint data.
  */
 public class CheckPointUserData implements java.io.Serializable {
```

Comment:
unrelated to your change, but wondering if we should follow the framework's precedence here and use `CheckpointUserData` rather than `CheckPointUserData`

Reply:
I have no preferences - which one ? you pick ;-)

Reply:
I vote to update any classnames that have `CheckPoint` and change them to `Checkpoint`